### PR TITLE
Added a question in python-quiz.ts

### PR DESCRIPTION
**Pull request by kalashjain23**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

#15 

Adding a question related to Dunder methods in python-quiz.ts file.
Also, I would like to make this contribution for the Hacktoberfest event.
Thank you!

### CODE REVIEW

File: src/data/python-quiz.ts [hunk 1]
```diff
@@ -1,4 +1,14 @@
 const pythonQuiz = [
+  {
+    Question: "In Python, which one of these is not a Dunder method?",
+    Answer: "__func",
+    Distractor1: "__init__",
+    Distractor2: "__add__",
+    Distractor3: "__len__ ",
+    Explanation:
+      "Dunder methods are names that are preceded and succeeded by double underscores, hence __func method is not a Dunder method.",
+    Link: "https://www.geeksforgeeks.org/dunder-magic-methods-python/",
```
Comment (jdwilkin4) [inline src/data/python-quiz.ts:10]: Please use a freecodecamp resource or official documentation.

For the freecodecamp resources, we accept articles, or  youtube videos.

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

File: src/data/python-quiz.ts (hunk 1 above)
Comment (kalashjain23) [reply src/data/python-quiz.ts]: I've made the required changes!
Thank you for reviewing!

**Review by jdwilkin4 — APPROVED** [review]
Thank you for your contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**